src/ansys/fluent/core/session.py

```diff
@@ -212,6 +210,11 @@ def __init__(self, _session):
         for obj in filter(None, (self._datamodel_events, self.transcript, self.events)):
             self._fluent_connection.register_finalizer_cb(obj.stop)
 
+    @property
+    def app_utilities(self):
```

Comment:
Maybe we don't need to expose this for users as this is used only in the implemntation code.

Reply:
Yes. Done. Thank you.